<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="{{url_for('static',filename='style.css')}}">
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link
        href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
        rel="stylesheet">
        <link rel="icon" href="{{ url_for('static', filename='images/icon.png') }}" type="image/png">
    <title>Parkinson Disease</title>
</head>

<body>
    <header>Parkinson's Gene based modelling</header>
    <main>
        <div class="top section">
            <div class="left sub_section">
                <span class="big_heading">Project Overview</span><br><br>
                Parkinson's disease (PD) is a progressive neurodegenerative disorder with complex genetic underpinnings.
                In this project, we leverage gene expression data from the <a href="https://www.ncbi.nlm.nih.gov/geo/"
                    target="_blank">Gene Expression Omnibus (GEO)</a>, specifically
                dataset <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54536"
                    target="_blank">GSE54536</a>, which comprises whole-transcriptome profiles of peripheral blood from
                untreated stage
                1 PD patients and healthy controls. Through differential gene expression analysis, functional
                enrichment, and gene interaction network construction, this research identifies key genes and pathways
                involved in PD pathogenesis, particularly focusing on endocytosis and alternative splicing mechanisms.
                The findings provide valuable insights into the molecular mechanisms of early-stage PD, offering
                potential targets for diagnostic and therapeutic strategies.
            </div>
            <div class="right sub_section">
                <span class="big_heading">Project Outline</span><br><br>
                <span class="small_heading" style="font-size: 1.3rem;font-weight: bold;">About Parkinson</span><br><br>
                <a href="https://en.wikipedia.org/wiki/Parkinson%27s_disease">Parkinson's disease (PD)</a> is a
                progressive neurodegenerative disorder that primarily affects movement,
                causing symptoms such as tremors, stiffness, and bradykinesia (slowness of movement). It is the second
                most common neurodegenerative disorder after Alzheimer's disease and primarily affects individuals over
                the age of 60. The exact cause of PD remains unclear, but it is thought to result from a combination of
                genetic and environmental factors. At the cellular level, the disease is characterized by the loss of
                dopamine-producing neurons in the brain, particularly in the substantia nigra, and the presence of Lewy
                bodiesâ€”abnormal aggregates of the protein alpha-synuclein. In addition to motor symptoms, PD often leads
                to cognitive and emotional changes, impacting a patient's overall quality of life. Understanding the
                genetic mechanisms underlying PD is critical for developing effective treatments and early diagnostic
                tools.<br><br>
                <span class="small_heading">Making the dataset</span><br><br>
                The dataset used for this project is derived from the Gene Expression Omnibus (GEO) and consists of the
                supplementary file named <b>GSE54536_Non-normalized_data.txt.gz</b>, which contains two key files. The
                first file, <b>GPL10558_HumanHT-12_V4_0_R1_15002873_B</b>, provides essential information about the
                genes,
                including Source, Gene names, Entrez IDs, Gene Ontology (GO) terms, and Probe IDs. The second file,
                <b>GSE54536_series_matrix.txt</b>, contains gene expression values associated with these Probe IDs. To
                create the final dataset for analysis, these two files were merged based on the Probe IDs, resulting in
                a comprehensive dataset for Parkinson's disease (PD). From this final dataset, key columns such as
                ILMN_Gene, Symbol, and the gene expression values for all samples were extracted for further analysis,
                allowing for a focused investigation into the gene expression profiles associated with PD.<br><br>
                <span class="small_heading">Data Analysis</span><br><br>
                <ul>
                    <li><strong>Gene Expression Visualization:</strong> Various plots were generated to visualize gene
                        expression profiles in Parkinson's disease (PD) patients compared to healthy controls,
                        highlighting upregulated and downregulated genes.</li>
                    <li><strong>ANOVA Analysis:</strong> An Analysis of Variance (ANOVA) was conducted to assess the
                        significance of gene expression differences, identifying genes with statistically significant
                        variations between PD and non-PD patients.</li>
                    <li><strong>Gene Ontology (GO) Enrichment Analysis:</strong> GO enrichment analysis was performed to
                        determine the biological processes and pathways that are overrepresented among differentially
                        expressed genes, providing insights into the underlying mechanisms of PD.</li>
                    <li><strong>Gene Interaction Network:</strong> A gene interaction network was constructed to explore
                        the relationships between the identified genes, enhancing the understanding of molecular
                        interactions contributing to the pathogenesis of Parkinson's disease.</li>
                    <li><strong>Overall Insights:</strong> This multifaceted analysis enhances the understanding of the
                        gene expression landscape in PD and highlights potential targets for further research and
                        therapeutic intervention.</li>
                </ul>
                <span class="small_heading">Note Section</span><br><br>


                In the dataset, there are a total of 47,231 genes; however, due to redundancy, it is important to create
                a subset of the data where genes are grouped, and their expression levels are summed. This approach
                allows for the calculation of the mean expression levels for both Parkinson's disease (PD) and non-PD
                samples. The following code demonstrates how to perform this procedure:

                <div class="code">
                    <pre>
    <b style="color: gray;">Group the dataset by gene identifier and sum the expression levels</b>
    <b style="color: rgb(239, 115, 115)">df_modify = df.groupby('ILMN_Gene', as_index=False).sum()</b>

    <b style="color: gray;">Calculate the mean expression levels for PD samples</b>
    <b style="color: rgb(6, 168, 92)">df_modify['Mean_PD'] = df_modify[['GSM1318547', 'GSM1318548', 'GSM1318549', 'GSM1318550',
    'GSM1318551']].mean(axis=1)</b>

    <b style="color: gray;">Calculate the mean expression levels for non-PD samples</b>
    <b style="color: rgb(53, 131, 203)">df_modify['MeanNon_PD'] = df_modify[['GSM1318552', 'GSM1318553', 'GSM1318554', 'GSM1318555',
    'GSM1318556']].mean(axis=1)</b>
                    </pre>

                </div>

                This code will create a modified DataFrame that includes the summed expression levels of genes and the
                calculated mean expression levels for both PD and non-PD samples.


            </div>
        </div>
        <div class="bottom section">
            <div class="result">
                <div class="image img1"></div>
                <div class="conclusion con1">
                    <h3>Box Plot</h3>
                    <ul>
                        <li><strong>Comparison:</strong> This box plot compares gene expression levels between two
                            groups: "Mean PD" and "Mean Non PD."</li>
                        <li><strong>Y-Axis Measurement:</strong> The y-axis measures gene expression from roughly -100
                            to over 300.</li>
                        <li><strong>Box Plot Components:</strong> Each box shows the interquartile range (IQR) of gene
                            expression levels, with the horizontal line inside the box indicating the median level.</li>
                        <li><strong>Whiskers and Outliers:</strong> Whiskers stretch to values within 1.5 times the IQR
                            from the lower and upper quartiles, and outliers are marked as individual points beyond the
                            whiskers.</li>
                        <li><strong>Mean Gene Expression:</strong> A green triangle inside each box likely marks the
                            mean gene expression level for each group.</li>
                        <li><strong>Insights:</strong> This plot helps in spotting differences or similarities in gene
                            expression for the groups.</li>
                        <li><strong>Findings:</strong> The plot suggests that the "Mean PD" group has higher median and
                            mean gene expression levels compared to the "Mean Non PD" group.</li>
                        <li><strong>Variation:</strong> There seems to be more variation within the "Mean PD" group, as
                            indicated by the larger interquartile range and more outliers.</li>
                        <li><strong>Implications:</strong> This could mean that the "Mean PD" group's gene expression is
                            more diverse or affected by different factors.</li>
                    </ul>

                </div>
                <div class="image img2"></div>
                <div class="conclusion con2">
                    <h3>Scatter Plot</h3>
                    <ul>
                        <li><strong>Comparison:</strong> This scatter plot shows the mean expression levels for
                            different samples.</li>
                        <li><strong>Sample Representation:</strong> PD samples are represented by red dots, while Non-PD
                            samples are represented by blue dots.</li>
                        <li><strong>X-Axis:</strong> The x-axis displays the sample numbers (1 to 10).</li>
                        <li><strong>Y-Axis:</strong> The y-axis shows the mean expression levels ranging from 6000 to
                            8000.</li>
                        <li><strong>Expression Levels:</strong> PD samples generally exhibit higher expression levels
                            compared to Non-PD samples.</li>
                        <li><strong>Utility:</strong> This visual is useful for comparing gene expression levels between
                            the two categories, hinting at biological differences.</li>
                    </ul>

                </div>
                <div class="image compare">
                    <ul>
                        <li><strong>Plots Analysis</strong>
                            <ul>
                                <li><strong>Box Plot:</strong>
                                    <ul>
                                        <li><strong>Comparison:</strong> Shows gene expression levels for "Mean PD" and
                                            "Mean Non PD."</li>
                                        <li><strong>Findings:</strong>
                                            <ul>
                                                <li><strong>"Mean PD" Group:</strong>
                                                    <ul>
                                                        <li>Higher median and mean expression levels.</li>
                                                        <li>Greater variability and more outliers.</li>
                                                    </ul>
                                                </li>
                                                <li><strong>"Mean Non PD" Group:</strong>
                                                    <ul>
                                                        <li>Lower expression levels.</li>
                                                        <li>More consistent across samples.</li>
                                                    </ul>
                                                </li>
                                            </ul>
                                        </li>
                                    </ul>
                                </li>
                                <li><strong>Scatter Plot:</strong>
                                    <ul>
                                        <li><strong>Overview:</strong> Displays mean expression levels of samples.</li>
                                        <li><strong>Outliers:</strong>
                                            <ul>
                                                <li>Sample 2 (PD) and Sample 7 (Non-PD) exhibit major differences from
                                                    the rest.</li>
                                                <li>The remaining samples show similar expression levels.</li>
                                            </ul>
                                        </li>
                                    </ul>
                                </li>
                                <li><strong>ANOVA Results:</strong>
                                    <ul>
                                        <li><strong>F-statistic:</strong> 1.04</li>
                                        <li><strong>P-value:</strong> 0.34</li>
                                    </ul>
                                </li>
                                <li><strong>Interpretation:</strong>
                                    <ul>
                                        <li><strong>Statistical Significance:</strong>
                                            <ul>
                                                <li>The P-value > 0.05 indicates no significant difference between the
                                                    means of PD and Non-PD groups.</li>
                                                <li>Visual differences, such as outliers, are not sufficient to conclude
                                                    a statistically significant difference.</li>
                                            </ul>
                                        </li>
                                    </ul>
                                </li>
                                <li><strong>Conclusion:</strong>
                                    <ul>
                                        <li>While visual plots suggest variations and specific outliers, ANOVA confirms
                                            that these differences are not statistically significant.</li>
                                        <li>These observed differences could be attributed to random variations,
                                            indicating consistency in gene expression across both groups, with some
                                            exceptions that warrant further investigation.</li>
                                    </ul>
                                </li>
                                <li><strong>Further Steps:</strong>
                                    <ul>
                                        <li>The non-significant differences may be related to false discovery rates,
                                            necessitating the implementation of the Benjamin-Hochberg test to identify
                                            genes that are truly significant.</li>
                                        <li>However, it is anticipated that the number of significant genes identified
                                            through this method will be minimal, making interpretation challenging.</li>
                                        <li>Therefore, we will consider all genes for further analysis to ensure a
                                            comprehensive understanding of the gene expression landscape in Parkinson's
                                            disease.</li>
                                    </ul>
                                </li>
                            </ul>
                        </li>
                    </ul>

                </div>
                <div class="image img3"></div>
                <div class="conclusion con3">
                    <h3>Volcano Plot</h3>
                    <ul>
                        <li><strong>Volcano Plot Overview:</strong> This plot visualizes gene expression data in the
                            context of Parkinson's Disease.</li>
                        <li><strong>X-axis (Log2 Fold Change):</strong>
                            <ul>
                                <li>Measures the magnitude of change in gene expression.</li>
                                <li>Positive values indicate upregulation (increase in expression).</li>
                                <li>Negative values indicate downregulation (decrease in expression).</li>
                            </ul>
                        </li>
                        <li><strong>Y-axis (-log10 p-value):</strong>
                            <ul>
                                <li>Indicates the statistical significance of changes in gene expression.</li>
                                <li>Higher values represent more significant changes.</li>
                            </ul>
                        </li>
                        <li><strong>Color-coded Dots:</strong>
                            <ul>
                                <li><strong>Red Dots:</strong> Represent upregulated genes with higher expression in the
                                    condition being studied.</li>
                                <li><strong>Blue Dots:</strong> Represent downregulated genes with lower expression.
                                </li>
                                <li><strong>Grey Dots:</strong> Represent genes that are not statistically significant.
                                </li>
                            </ul>
                        </li>
                        <li><strong>Horizontal Red Dashed Line:</strong>
                            <ul>
                                <li>Set around y = 1.3, marking the threshold for statistical significance (p-value =
                                    0.05).</li>
                                <li>Genes above this line are considered statistically significant.</li>
                            </ul>
                        </li>
                        <li><strong>Significance:</strong>
                            <ul>
                                <li>This plot highlights significantly upregulated and downregulated genes in
                                    Parkinson's Disease.</li>
                                <li>It pinpoints potential areas for further research or therapeutic targeting.</li>
                            </ul>
                        </li>
                    </ul>

                </div>
                <div class="image img4"></div>
                <div class="conclusion con4">
                    <h3>MA Plot</h3>
                    <ul>
                        <li><strong>X-axis (A)</strong>: Represents the average expression level of genes, allowing for
                            a direct comparison of gene activity across samples.</li>
                        <li><strong>Y-axis (M)</strong>: Indicates the log2 fold change in expression levels, which
                            helps identify genes that are significantly upregulated or downregulated.</li>
                        <li><strong>Colors and Markers:</strong>
                            <ul>
                                <li><strong>Red dots</strong>: Represent upregulated genes, indicating a significant
                                    increase in expression levels associated with the condition under study.</li>
                                <li><strong>Blue dots</strong>: Indicate downregulated genes, showing a notable decrease
                                    in expression that may relate to disease mechanisms.</li>
                                <li><strong>Gray dots</strong>: Highlight genes with no significant change in
                                    expression, serving as a reference for unaffected genes in the dataset.</li>
                            </ul>
                        </li>
                        <li><strong>Interpretation:</strong>
                            <ul>
                                <li>Points above the horizontal line (y=0) signify upregulated genes (red), while points
                                    below this line indicate downregulated genes (blue).</li>
                                <li>The spread of points illustrates the variability in gene expression changes across
                                    the dataset, with a wider spread suggesting more pronounced differences.</li>
                                <li>A central cluster near the horizontal line represents genes with little to no change
                                    in expression, indicating a baseline level of activity that remains stable.</li>
                                <li>This plot is instrumental in pinpointing which genes are significantly affected in
                                    the context of Parkinson's Disease, providing insights into potential biological
                                    pathways and mechanisms at play.</li>
                            </ul>
                        </li>
                    </ul>
                </div>
                <div class="image compare">
                    <h3>Upregulated Genes Analysis</h3>
                    <ul>
                        <li>After performing the Benjamini-Hochberg test, several key upregulated genes were identified
                            in Parkinson's Disease (PD):</li>
                        <ul>
                            <li><strong>OSBPL10</strong>: Indicates heightened immune responses.</li>
                            <li><strong>HLA-DQA1</strong>: Suggests involvement in inflammatory processes.</li>
                            <li><strong>TLR10</strong>: Reflects alterations in lipid metabolism.</li>
                            <li><strong>CD24</strong>: Associated with changes in cell signaling and communication.</li>
                            <li><strong>CXCR4</strong>: May contribute to neuronal survival and adaptation mechanisms.
                            </li>
                        </ul>
                        <li>These upregulated genes suggest an inflammatory environment and cellular adaptations in
                            response to stress or damage in PD.</li>
                    </ul>

                    <h3>Downregulated Genes Analysis</h3>
                    <ul>
                        <li>The Benjamini-Hochberg test revealed significant downregulation of genes in PD:</li>
                        <ul>
                            <li><strong>LOC92017</strong>: Implicates reduced neuronal plasticity.</li>
                            <li><strong>FOS</strong>: Suggests compromised cellular functions critical for neuronal
                                health.</li>
                            <li><strong>PKD1</strong>: Reflects a loss of protective mechanisms within neurons.</li>
                            <li><strong>RRP12</strong>, <strong>LOC642393</strong>, <strong>NPCDR1</strong>,
                                <strong>KIAA1530</strong>, and <strong>JSRP1</strong>: Additional genes indicating
                                similar trends in reduced expression.
                            </li>
                        </ul>
                        <li>This downregulation points to impairments in the ability of neurons to cope with stressors
                            associated with PD, emphasizing potential targets for therapeutic intervention.</li>
                    </ul>

                </div>
                <div class="image img5"></div>
                <div class="conclusion con5">
                    <h3>GO Enrichment Analysis Bar Chart Interpretation</h3>
                    <ul>
                        <li><strong>X-axis</strong>: Represents the counts of genes associated with specific Gene
                            Ontology (GO) terms, illustrating the prevalence of genes linked to particular biological
                            functions.</li>
                        <li><strong>Y-axis</strong>: Displays various GO terms categorized into three types:
                            <ul>
                                <li><strong>Orange bars</strong>: Represent Biological Process (BP) terms, such as
                                    "regulation of cellular processes" and "protein binding." These processes are
                                    crucial in PD as they can affect neuronal health and communication, influencing
                                    disease progression.</li>
                                <li><strong>Blue bars</strong>: Indicate Cellular Component (CC) terms, including
                                    "cytoplasm" and "membrane." Changes in cellular components can affect cell signaling
                                    and interactions, playing a vital role in neuronal function and survival.</li>
                                <li><strong>Green bars</strong>: Reflect Molecular Function (MF) terms, like "small
                                    molecule binding" and "ion binding." These functions are significant in PD since
                                    they can impact neurotransmitter dynamics and cellular homeostasis, ultimately
                                    affecting motor control and cognitive functions.</li>
                            </ul>
                        </li>
                        <li><strong>Bar Length</strong>: Each bar's length represents the number of genes associated
                            with that particular GO term, with black dots indicating the exact counts. Longer bars
                            suggest a higher number of genes involved in specific biological activities related to PD.
                        </li>
                        <li><strong>Insights</strong>: The enriched GO terms highlight key areas of genetic activity
                            relevant to Parkinson's Disease. By focusing on processes, components, and functions that
                            are significantly represented, this analysis provides a clearer picture of the biological
                            mechanisms potentially underlying PD pathology and identifies pathways that could be
                            targeted for therapeutic interventions.</li>
                    </ul>


                </div>
                <div class="image img6">
                    <div class="subset_network">Subset Network 1</div>
                </div>
                <div class="image img7">
                    <div class="subset_network">Subset Network 2</div>
                </div>
                <div class="conclusion con6">
                    <h3>Gene Interaction Network Interpretation</h3>
                    <ul>
                        <li><strong>Nodes</strong>: Each node represents a gene within the network, providing a visual
                            representation of gene relationships.</li>
                        <li><strong>Red nodes</strong>: Indicate upregulated genes, suggesting increased expression
                            levels that may play a significant role in the biological processes associated with
                            Parkinson's Disease (PD).</li>
                        <li><strong>Blue nodes</strong>: Represent downregulated genes, highlighting areas where gene
                            expression is reduced, which can also impact cellular functions and contribute to PD
                            pathology.</li>
                        <li><strong>Gray nodes</strong>: These are neighboring genes that are not significantly
                            upregulated or downregulated, serving as a backdrop that helps to identify potential
                            interactions involving non-significant genes.</li>
                        <li><strong>Edges</strong>: The lines connecting the nodes illustrate the interactions between
                            genes, providing insights into the relationships and regulatory networks that may be
                            influential in disease processes.</li>
                        <li><strong>Insights</strong>:
                            <ul>
                                <li>The density of connections can indicate which genes are central to the network,
                                    potentially influencing multiple biological pathways related to PD.</li>
                                <li>The color coding allows for a quick assessment of which genes are upregulated or
                                    downregulated, facilitating the identification of key targets for further
                                    investigation.</li>
                                <li>Gray nodes provide context for understanding the broader network, revealing how
                                    non-significant genes are connected to the more actively expressed ones.</li>
                            </ul>
                        </li>
                        <li><strong>Limitations</strong>: Given that there are over 10,000 upregulated and downregulated
                            genes identified in this analysis, it is impractical to display the full interaction
                            network. Instead, the two networks presented are randomly selected subsets, allowing for a
                            focused examination of gene interactions while acknowledging the complexity and richness of
                            the full dataset.</li>
                    </ul>

                </div>
                <div class="image compare">
                    <h3>Upregulated Genes Analysis</h3>
                    <ul>
                        <li>After performing the Benjamini-Hochberg test, several key upregulated genes were identified
                            in Parkinson's Disease (PD):</li>
                        <ul>
                            <li><strong>OSBPL10</strong>: Indicates heightened immune responses.</li>
                            <li><strong>HLA-DQA1</strong>: Suggests involvement in inflammatory processes.</li>
                            <li><strong>TLR10</strong>: Reflects alterations in lipid metabolism.</li>
                            <li><strong>CD24</strong>: Associated with changes in cell signaling and communication.</li>
                            <li><strong>CXCR4</strong>: May contribute to neuronal survival and adaptation mechanisms.
                            </li>
                        </ul>
                        <li>These upregulated genes suggest an inflammatory environment and cellular adaptations in
                            response to stress or damage in PD.</li>
                    </ul>

                    <h3>Downregulated Genes Analysis</h3>
                    <ul>
                        <li>The Benjamini-Hochberg test revealed significant downregulation of genes in PD:</li>
                        <ul>
                            <li><strong>LOC92017</strong>: Implicates reduced neuronal plasticity.</li>
                            <li><strong>FOS</strong>: Suggests compromised cellular functions critical for neuronal
                                health.</li>
                            <li><strong>PKD1</strong>: Reflects a loss of protective mechanisms within neurons.</li>
                            <li><strong>RRP12</strong>, <strong>LOC642393</strong>, <strong>NPCDR1</strong>,
                                <strong>KIAA1530</strong>, and <strong>JSRP1</strong>: Additional genes indicating
                                similar trends in reduced expression.</li>
                        </ul>
                        <li>This downregulation points to impairments in the ability of neurons to cope with stressors
                            associated with PD, emphasizing potential targets for therapeutic intervention.</li>
                    </ul>

                    <h3>Conclusion</h3>
                    <ul>
                        <li>The analysis of upregulated and downregulated genes provides critical insights into the
                            molecular landscape of PD. The enriched Gene Ontology (GO) terms associated with these genes
                            highlight significant biological processes, cellular components, and molecular functions
                            that are altered in PD:</li>
                        <ul>
                            <li>Upregulated genes are linked to immune responses and inflammation, suggesting that these
                                pathways are crucial in the pathology of PD.</li>
                            <li>Downregulated genes indicate potential impairments in neuronal health and plasticity,
                                further illustrating how these genes are integral to understanding PD mechanisms.</li>
                        </ul>
                        <li>Overall, these findings underscore the importance of the identified genes and their
                            associated GO terms in impacting the progression of PD, offering promising avenues for
                            future research and potential therapeutic strategies.</li>
                    </ul>

                </div>

            </div>
        </div>
    </main>
    <footer> Anurupa Karmakar &copy; 2023 </footer>
</body>

</html>